docs/source/tutorial/en/vectorization.rst

```diff
@@ -0,0 +1,157 @@
+Vectorization
+==================
+
+This section provides a simple interface to convert the incoming items into vectors directly. Currently, the option of whether to use the pre training model is provided. You can choose according to your needs. If you don't want to use the pre-trained model, you can call D2V directly, or call get_pretrained_i2v function if you want to use the pre-trained model.
+
+- Don't use the pre-trained model
+
+- Use the pre-trained model
+
+Overview Flow
+---------------------------
+
+1.Perform `syntax parsing <parse.rst>`_ on incoming items to get items in SIF format；
+
+2.Perform `component segmentation <seg.rst>`_ on sif_items;
+
+3.Perform `tokenization <tokenize.rst>`_ on segmented items;
+
+4.Use the existing or pre-trained model we provided to convert the tokenized items into vectors.
+
+
+Don't use the pre-trained model: call existing models directly
```

Comment:
Change the order seems better:  
- 1. Use the pre-training model: call get_pretrained_i2v directly
- 2. Don't use the pre-trained model: call existing models directly
(and please adjust the Chinese counterpart in tutorial/zh/vectorization.rst)